scripts/start.js

```diff
@@ -16,6 +16,9 @@ var WebpackDevServer = require('webpack-dev-server');
 var config = require('../config/webpack.config.dev');
 var execSync = require('child_process').execSync;
 var opn = require('opn');
+var portfinder = require('portfinder');
+
```

Comment:
I think you can have a `var PORT = 3000` here and let the callback of `portfinder.getPort` re-assign this variable. Then the webpack `done` callback can just do `console.log('The app is running at http://localhost:' + PORT + '/');`